src/components/common/textLines/TextWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { type FC, type PropsWithChildren } from 'react';
+import classNames from 'classnames';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+
+interface TextWrapperProps {
+    index?: number;
+    isLargeStyle?: boolean;
+    className?: string;
+}
+
+const TextWrapper: FC<PropsWithChildren<TextWrapperProps>> = ({
+    index,
+    isLargeStyle,
+    className,
+    children
+}) => {
+    if (index === 0) {
```

Comment:
How about adding an `isHeading` or `isPrimary` prop and moving the `index` logic to the `TextLines` component?